bechdel-test/bechdel.py

```diff
@@ -10,7 +10,7 @@
 
 ## Revenue Data
 numbers_df = pd.read_csv('revenue.csv',encoding='utf8',index_col=0)
-numbers_df['Released'] = pd.to_datetime(numbers_df['Released'],unit='D')
+numbers_df['Released'] = pd.to_datetime(numbers_df['Released'])
```

Comment:
I had already changed it back (check [here](https://github.com/ViDA-NYU/reprozip-examples/pull/6/commits/53a452231632ffa5e21fc524d9e197d06cb00372)). Do you happen to know why this is not being reflected? Perhaps when I moved the commits to a new branch I may have done something wrong?